contracts/0.4.24/oracle/LidoOracle.sol

```diff
@@ -30,7 +32,7 @@ import "./ReportUtils.sol";
  * Not all frames may come to a quorum. Oracles may report only to the first epoch of the frame and
  * only if no quorum is reached for this epoch yet.
  */
-contract LidoOracle is AragonApp {
+contract LidoOracle is AragonApp, Versioned {
```

Comment:
This contract is being re-written from scratch with the absolute most of the code removed.